android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java

```diff
@@ -212,9 +224,9 @@ public void setConfiguration(PlayerView view, ReadableMap config) {
     if (configuration != null && configuration.getString("url") != null) {
       url = configuration.getString("url");
 
-      if (configuration.getString("subtitles") != null) {
-        subtitleTrack = new SubtitleTrack(configuration.getString("subtitles"), null, "en", "en", false, "en");
-      }
+      //if (configuration.getString("subtitles") != null) {
```

Comment:
As above

Reply:
it is a WIP, about subtitles. I will remove it later.